src/main/java/org/session5/homework/FibonacciSeriesGenerator.java

```diff
@@ -0,0 +1,27 @@
+package org.session5.homework;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FibonacciSeriesGenerator {
+
+    private FibonacciSeriesGenerator() {
+        throw new IllegalStateException("Fibonacci Series Generator");
+    }
+
+    public static ArrayList<Integer> generateTheFibonacciSeries(int firstNNumbersToGenerate) {
+        List<Integer> fibonacciSeries = new ArrayList<>();
+        switch (firstNNumbersToGenerate) {
+            case 0 -> fibonacciSeries.add(0);
+            case 1 -> fibonacciSeries.add(1);
+            default -> {
+                fibonacciSeries.addAll(Arrays.asList(0, 1));
+                for (int index = 2; index <= firstNNumbersToGenerate; index++) {
+                    fibonacciSeries.add(index, fibonacciSeries.get(index - 1) + fibonacciSeries.get(index - 2));
+                }
+            }
+        }
+        return (ArrayList<Integer>) fibonacciSeries;
```

Comment:
It's better to use the interface List<Integer> rather than the concrete implementation ArrayList<Integer> for the method return type.